packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.test.js

```diff
@@ -128,101 +134,357 @@ describe('overrides plugin', () => {
+        overridesResolver.handleHook(testRequestContext, {}, callback, resolver)
+
+        const nestedRewriteDir = 'src/configs/webpack/test/path/to/overrides'
+
+        expect(callback).toHaveBeenCalled()
+        expect(resolver.doResolve).toHaveBeenCalledWith(
+            null,
+            createRequestContextWith({
+                path: path.join(nestedRewriteDir, REQUEST_PATH + REQUEST_EXTENSION),
+                request: `${EXTENDS_TARGET}/${REQUEST_PATH}`
+            }),
+            expect.anything(),
+            expect.anything(),
+            expect.anything()
+        )
+    })
+})
+
+describe('OverridePlugin.isFromExtends', () => {
+    let os
+    let opts = options
+    let cases
+
+    if (path.sep === '\\') {
+        // WINDOWS test cases
+        os = 'windows'
+
+        opts = {
+            overridesDir: '\\overrides',
+            extends: ['@salesforce/retail-react-app'],
+            projectDir: `src\\configs\\webpack\\test`
+        }
+
+        cases = [
+            {
+                name: 'Import from a package in extends',
+                request: '@salesforce/retail-react-app/exists',
+                filepath: '.\\node_modules\\@salesforce\\retail-react-app',
+                result: true
+            },
+            {
+                name: 'Import from a package not in extends',
+                request: '@salesforce/express-minimal/notExists',
+                filepath: '.\\node_modules\\@salesforce\\retail-react-app',
+                result: false
+            },
+            {
+                name: 'Do not trigger override if filepath of issuer contains overrides directory',
+                request: '@salesforce/retail-react-app/exists',
+                filepath: 'src\\configs\\webpack\\test\\overrides\\exists',
+                result: false
+            }
+        ]
+    } else {
+        // POSIX test cases
+        os = 'posix'
+
+        cases = [
+            {
+                name: 'Import from a package in extends',
+                request: '@salesforce/retail-react-app/exists',
+                filepath: './node_modules/@salesforce/retail-react-app',
+                result: true
+            },
+            {
+                name: 'Import from a package not in extends',
+                request: '@salesforce/express-minimal/notExists',
+                filepath: './node_modules/@salesforce/retail-react-app',
+                result: false
+            },
+            {
+                name: 'Do not trigger override if filepath of issuer contains overrides directory',
+                request: '@salesforce/retail-react-app/exists',
+                filepath: 'src/configs/webpack/test/overrides/exists',
+                result: false
+            }
+        ]
+    }
+
+    const plugin = new OverridesResolverPlugin(opts)
+
+    describe('Testing scenarios for: ' + os, () => {
+        cases.forEach((c) =>
```

Comment:
nit, single character variables are not readable and difficult to do future find + replaces on, let's use something like `case` or anything with more chars

Reply:
c has been renamed to `testCase` since `case` is a keyword and is not available